### DISCUSSION TITLE
DynamoDBLocal in v2

### DISCUSSION THREAD

**Discussion by atmakur** [aws/aws-sdk-java-v2#4679] [General]
Hi,

Do we have an alternative in v2 for:

```
import com.amazonaws.services.dynamodbv2.local.main.ServerRunner
import com.amazonaws.services.dynamodbv2.local.server.DynamoDBProxyServer

DynamoDBProxyServer server = ServerRunner.createServerFromCommandLineArgs(... args here );
server.start();
```

Can someone help me with this?

**Comment by debora-ito** [comment]
@atmakur DynamoDBLocal is not maintained by the Java SDK team. You can reach out to the DynamoDBLocal team here: https://github.com/awslabs/amazon-dynamodb-local-samples/issues

**Reply by atmakur** [reply]
Thank you @debora-ito
